### DISCUSSION TITLE
Remove Edit URL from Meeting Feedback (TSMLui)

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#875] [Q&A]
Can the edit URL be removed from TSML UI meeting feedback email? 

Thanks.

**Comment by Grateful-Viv** [comment]
Can this be addressed please? Thanks.

**Reply by tim-rohrer** [reply]
Are you referring to the email generated from this link?

<img width="413" alt="image" src="https://user-images.githubusercontent.com/7756865/182244969-3760da16-e1f8-4f8c-9c0e-1217ee76003b.png">

What is the issue with the link being present? It makes a quick and easy way for the admin to copy & paste and access the meeting for making changes.

**Reply by Grateful-Viv** [reply]
It's fine to have the public link, but why does it also include the edit link? The admin can copy and paste the meeting URL given from the first URL.

This is an example of what is coming through and I am suggesting (or asking) if the Edit URL does not have to be included.

​​Public URL: https://sussexaa.org/meetings/?meeting=lewes-beach-meeting
​​Edit URL: https://sussexaa.org/wp-admin/post.php?post=5017&action=edit

**Comment by tim-rohrer** [comment]
Without a compelling reason to remove the link, I do not see us making this change.

Including the link makes it easier for the admin (for whom the email is being generated) to navigate directly to the meeting for which the feedback is being requested. One click, log in, and then make any desired changes to the meeting. This reduces the guesswork and limits the chances of making an error.

**Reply by Grateful-Viv** [reply]
The compelling reason is it confuses the person who is sending the email with unhelpful and unnecessary information. 

One click to the public URL would also give the admin the ability to make the needed changes. There should be no guesswork or chances for error when admin clicks the Public URL. It's exactly the same as if they clicked the Edit URL.

**Reply by tim-rohrer** [reply]
With my tests, clicking on the public URL does not bring up the edit meeting page. Clicking on the edit meeting link did bring me to the edit meeting page.

**Reply by Grateful-Viv** [reply]
You can also click public url and then edit meeting if you’re an admin.
